### PR TITLE
WOR-47: Terms of Service feature branch

### PR DESCRIPTION
**Pull request by MatthewBemis**
Ticket: <Link to Jira ticket>
<Put notes here to help reviewer understand this PR>

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've filled out the [Security Risk Assessment](https://broadinstitute.github.io/dsp-appsec-security-risk-assessment/) and attached the result to the JIRA ticket

### CODE REVIEW

**Comment by MatthewBemis** [comment]
jenkins retest

**Comment by MatthewBemis** [comment]
jenkins retest

**Review by aherbst-broad — APPROVED** [review]
I reviewed the constituent PRs for this feature branch

File: src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV1Spec.scala [hunk 1]
```diff
@@ -42,23 +43,23 @@ class UserRoutesV1Spec extends UserRoutesSpecHelper{
...
-      res.enabled shouldBe Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true, "tosAccepted" -> true)
+      res.enabled shouldBe Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true, "tosAccepted" -> true, "adminEnabled" -> true)
     }
   }
 
   it should "forbid the registration if ToS is enabled and the user doesn't specify the correct ToS url" in {
-    val (user, _, routes) = createTestUser(tosEnabled = true, tosAccepted = false)
+    val (_, _, routes) = createTestUser(tosEnabled = true, tosAccepted = false)
 
     val tos = TermsOfServiceAcceptance("onemillionpats.com")
```
Comment (marctalbott) [inline src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV1Spec.scala:61]: This should really count as a correct ToS url

**Review by marctalbott — APPROVED** [review]

**Pull request merged**